### ISSUE TITLE
Recursive LineStyle Error

### ISSUE THREAD

**Issue by danyeaw** [gaphor/gaphor#829]
**Describe the bug**

A head of a generalization relationship in the STPA example model was disconnected. When I try to reconnect it I am getting a recursive AttributeError:

```
Traceback (most recent call last):
  File "/home/dan/Projects/gaphor/gaphor/transaction.py", line 29, in _transactional
    return func(*args, **kwargs)
  File "/home/dan/Projects/gaphor/gaphor/diagram/propertypages.py", line 291, in _on_orthogonal_change
    self.horizontal_button.set_sensitive(active)
AttributeError: 'LineStylePage' object has no attribute 'horizontal_button'
```

**To Reproduce**
1. Open stpa.gaphor
2. Open UCAs, Hazards and Losses diagram in the package in the same name
3. Try to reconnect the generalization

**Expected behavior**
Generalization reconnects correctly.

**Screenshots**
![image](https://user-images.githubusercontent.com/10014976/119429156-4fa56b00-bcdc-11eb-913a-8d90d8d27cd3.png)

**OS**
- [X] Linux (Please put in notes the specific distro)
- [ ] macOS
- [ ] Windows

NOTES:

**Version**
Version of Gaphor: master

**Additional context**

**Comment by amolenaar** [comment]
Is this still an issue on master as well? I fixed it in 142a167081c7052575a4a0f4d0a4608f588f9958, I think.

**Comment by danyeaw** [comment]
Thanks @amolenaar, looks like it is now fixed!
